lms/djangoapps/verify_student/views.py

```diff
@@ -724,7 +731,9 @@ def create_order(request):
         return HttpResponseBadRequest(_("No selected price or selected price is below minimum."))
 
     if current_mode.sku:
-        return create_order_with_ecommerce_service(request.user, course_id, current_mode)
+        return create_order_with_ecommerce_service(
+            request.user, course_id, current_mode, request.POST.get('processor', settings.CC_PROCESSOR_NAME)
```

Comment:
I don't think it's necessary to default to `settings.CC_PROCESSOR_NAME` in the event that no processor name is included in the request. `create_order_with_ecommerce_service()` uses the API client's `create_basket()` under the hood, to which `payment_processor` is an optional argument defaulting to `None`. On the Otto side, if a payment processor name is required but none is provided, the default payment processor (specified in configuration) is used.